themes/src/main/resources/theme/keycloak/login/theme.properties

```diff
@@ -148,8 +148,6 @@ kcLogoIdP-gitlab=fa fa-gitlab
 kcLogoIdP-paypal=fa fa-paypal
 kcLogoIdP-stackoverflow=fa fa-stack-overflow
 kcLogoIdP-twitter=fa fa-twitter
-kcLogoIdP-openshift-v4=pf-icon pf-icon-openshift
-kcLogoIdP-openshift-v3=pf-icon pf-icon-openshift
```

Comment:
This ends up having no icon for OpenShift IdP I pressume? Don't think that's the end of the world, but wouldn't this be available in PF4?

Reply:
Yeah, I was a bit too eager to remove this. I've restored this for now, perhaps we can get rid of it in the future.